testbeds/web/example.html

```diff
@@ -217,9 +217,10 @@
         description: 'Branch Demo app!',
         openAppButtonText: 'Open',
         downloadAppButtonText: 'Download',
-        iframe: true,     // Show banner in an iframe if CSS on your page is conflicting
-        showMobile: true, // true by default, just set here for an example
-        showDesktop: true // true by default, just set here for an example
+        iframe: true,       // Show banner in an iframe if CSS on your page is conflicting
+        showiOS: true,      // true by default, just set here for an example
+        showDesktop: true,  // true by default, just set here for an example
+        showiPad: true      // true by default, just set here for an example
```

Comment:
Yeah, play around with these here. I made a four by four grid of every combination of showiPad and show iOS. Then tested each of those in iPad, and iPhone, for a total of 16 tests.